webview-ui/src/components/settings/ModelPicker.tsx

```diff
@@ -58,18 +52,14 @@ export const ModelPicker = ({
 		},
 		[modelIdKey, modelInfoKey, models, setApiConfigurationField, defaultModelInfo],
 	)
-
 	const inputValue = apiConfiguration[modelIdKey]
-
 	useEffect(() => {
 		if (!inputValue && !isInitialized.current) {
 			const initialValue = modelIds.includes(selectedModelId) ? selectedModelId : defaultModelId
-			setApiConfigurationField(modelIdKey, initialValue)
+			setApiConfigurationField(modelIdKey as any, initialValue)
 		}
-
 		isInitialized.current = true
 	}, [inputValue, modelIds, setApiConfigurationField, modelIdKey, selectedModelId, defaultModelId])
-
```

Comment:
Maybe also completely back this file out of the diff? I don't think any of the changes here are necessary.